appcenter-data/Data.js

```diff
@@ -50,10 +52,43 @@ function read(documentId, partition, readOptions) {
  * If the operation fails, the promise is rejected with an exception containing the details of the error.
  */
 function create(documentId, document, partition, writeOptions) {
-    if (writeOptions === undefined) {
+    if (!writeOptions) {
         writeOptions = new Data.WriteOptions(TimeToLive.DEFAULT);
     }
-    return AppCenterReactNativeData.create(documentId, document, partition, writeOptions).then(convertTimestampToDate);
+    return AppCenterReactNativeData.create(documentId, partition, document, writeOptions).then(convertTimestampToDate);
+}
+
+/**
+ * Delete a document.
+ *
+ * @param {string} documentId - The CosmosDB document id.
+ * @param {string} partition - The CosmosDB partition key.
+ * @param {object} writeOptions - Cache write options when the operation is done offline.
+ * @return {Promise} Future asynchronous operation with result being the document with metadata.
+ * If the operation fails, the promise is rejected with an exception containing the details of the error.
+ */
+function remove(documentId, partition, writeOptions) {
+    if (!writeOptions) {
```

Comment:
Please do not revert that as it would match 0 or false.
On typescript we will be more strict in the definition also.
null is pass through to native SDK which allows it, so it's actually better to check for undefined.

Reply:
Done